<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>251</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>영상 불러오기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lbl_status">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>600</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>660</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>570</y>
     <width>251</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>출력 폴더 지정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>570</y>
     <width>251</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>이미지 추출</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>230</y>
     <width>271</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>2. Crop 정보 : </string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="edit_h_2"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="edit_y"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>t(s) :</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>w :</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>h :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>x :</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>y :</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="edit_t"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>rgb :</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="edit_rgb"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="edit_w_2"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edit_x"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>30</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="edit_fps"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>h : </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>w : </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>1. 영상 정보 :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edit_w"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>fps :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edit_h"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
